main/main.cpp

```diff
@@ -518,8 +519,9 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("-e, --editor", "Start the editor instead of running the scene.\n", CLI_OPTION_AVAILABILITY_EDITOR);
 	print_help_option("-p, --project-manager", "Start the project manager, even if a project is auto-detected.\n", CLI_OPTION_AVAILABILITY_EDITOR);
 	print_help_option("--debug-server <uri>", "Start the editor debug server (<protocol>://<host/IP>[:port], e.g. tcp://127.0.0.1:6007)\n", CLI_OPTION_AVAILABILITY_EDITOR);
+	print_help_option("--dap-port <port>", "Use the specified port for the GDScript Debugger Adaptor protocol. Recommended port range [1024-49151].\n", CLI_OPTION_AVAILABILITY_EDITOR);
 #if defined(MODULE_GDSCRIPT_ENABLED) && !defined(GDSCRIPT_NO_LSP)
-	print_help_option("--lsp-port <port>", "Use the specified port for the GDScript language server protocol. The port should be between 1025 and 49150.\n", CLI_OPTION_AVAILABILITY_EDITOR);
+	print_help_option("--lsp-port <port>", "Use the specified port for the GDScript language server protocol. Recommended port range [1024-49151].\n", CLI_OPTION_AVAILABILITY_EDITOR);
```

Comment:
Shouldn't the range be `[1025-49150]`, if we go by the usual meaning of `[ ]` meaning inclusive the bounds?

@Calinou wrote:
https://github.com/godotengine/godot/pull/36252#issuecomment-1732020974
> Port numbers below 1025 can only be opened by root on Linux/macOS, and port numbers above 49150 are reserved for client ports ("ephemeral" ports).

Reply:
I was relying on the following info:

Well-known/System Ports: 0 – 1023
Registered Ports: 1024 – 49151
Dynamic/Private Ports: 49152 – 65535

from https://www.stationx.net/common-ports-cheat-sheet/

Reply:
Yes, I have used square brackets [ ] math notation to denote an interval including the bounds.

Reply:
For this notation I'd say it should be:
```suggestion
	print_help_option("--lsp-port <port>", "Use the specified port for the GDScript language server protocol. Recommended port range [1024, 49151].\n", CLI_OPTION_AVAILABILITY_EDITOR);
```